lib/server/index.js

```diff
@@ -223,6 +227,11 @@ Server.prototype = {
     var config = this.config;
     var test_page = config.get('test_page');
 
+    // If there is an array of test pages, we cycle through them as the default
+    if (Array.isArray(test_page)) {
+      test_page = test_page[this.testPageCount % test_page.length];
+    }
```

Comment:
I would forward always to the first page in this case, as randomly sending launchers might hide the broken behaviour that not every launcher is really testing every batch.